install/running_install.adoc

```diff
@@ -655,9 +658,12 @@ intended. However, this is a transient event and the system does correct itself
 over time. See https://github.com/kubernetes/kubernetes/issues/10030 for
 details.
 
-- On failure of the Ansible installer, you must start from a clean operating
-system installation. If you are using virtual machines, start from a fresh
-image. If you are using bare metal machines, see
+- If the Ansible installer fails and you did not need to change a host name or
+generate new certificates, run the *_deploy_cluster.yml_*
+playbook again. If you changed a host name, generated new certificates, or the
+installer fails again, start over with a clean operating system installation.
```

Comment:
In addition to start over with a clean operating system installation, it would also be feasible to run the uninstall playbook.